common/data/content-format-ids.ts

```diff
@@ -1,11 +1,13 @@
 export const ArticleFormatIds = {
-  ImageGallery: 'W5uKaCQAACkA3C0T',
-  // Essay seems to signify an article that is part of a series
-  Essay: 'W7TfJRAAAJ1D0eLK',
+  ImageGallery: 'W5uKaCQAACkA3C0T', // Should this be called In Pictures?  https://wellcomecollection.prismic.io/documents~k=article-formats&b=working&c=published&l=en-gb/W5uKaCQAACkA3C0T/
+  Article: 'W7TfJRAAAJ1D0eLK',
   Comic: 'W7d_ghAAALWY3Ujc',
   Podcast: 'XwRZ6hQAAG4K-bbt',
   BookExtract: 'W8CbPhEAAB8Nq4aG',
   LongRead: 'YxcjgREAACAAkjBg',
+  ProsePem: 'YrwCTxEAACUALJQ0',
```

Comment:
Do we do anything with these new ones? I'm thinking we should only have the ones we need, i.e. if we display things differently based on type.

Reply:
Nah, I can remove them, just thought I'd add the missing ones but if they're not useful I'll remove!